spec/facility_spec.rb

```diff
@@ -23,4 +29,11 @@
       expect(@facility.services).to eq(['New Drivers License', 'Renew Drivers License', 'Vehicle Registration'])
     end
   end
+
+  describe "#register_vehicle" do
+    it "can register a vehicle" do
+      expect(@facility.registered_vehicle).to eq(Vehicle.new)
+    end
```

Comment:
We don't want this method to return a vehicle object, but rather an array of vehicle objects. In this case though, we haven't added any vehicles, so it should return an empty array.